runtime/v2/builder.go

```diff
@@ -62,14 +64,6 @@ func (a *AppBuilder[T]) RegisterModules(modules map[string]appmodulev2.AppModule
 	return nil
 }
 
-// RegisterStores registers the provided store keys.
-// This method should only be used for registering extra stores
-// which is necessary for modules that not registered using the app config.
-// To be used in combination of RegisterModules.
-func (a *AppBuilder[T]) RegisterStores(keys ...string) {
```

Comment:
How do you do this then? This was essential to register non depinject modules

Reply:
You'd call `RegisterKey` on StoreBuilder instead

Reply:
Can add some docs tomorrow when I'm doing other docs PRs